inst/extdata/config/concept-dict/respiratory.json

```diff
@@ -202,6 +200,13 @@
                         "1006"
                     ],
                     "sub_var": "itemid"
+                },
+                {
+                    "table": "surgery_vital_signs",
```

Comment:
did you discuss with martin with regards to the surgery variables? Did you do this everywhere?

Reply:
Yes, he said to integrate them and it makes sense to include them all. I added all surgery variables, also the cases where we have previously skipped them.